nixos/modules/services/web-apps/immich.nix

```diff
@@ -182,6 +182,17 @@ in
         // {
           default = true;
         };
+      enableVectorChord =
+        mkEnableOption "the new VectorChord extension for full-text search in Postgres"
+        // {
+          default = true;
```

Comment:
Should this be enabled by default?

Reply:
I think it's okay to have it on by default as I haven't heard of any breakage but I guess it goes against 25.05 being "stable".

Reply:
@dotlambda I also haven't heard of anything breaking, and having this on would help with migration of users that don't read release notes.

However, I'm fine with having this disabled for now and making it `default = true;` in a few weeks.
Let me know what's preferred

Reply:
I think it should be off by default to not cause a backwards-incompatible state change in Immich.

I think we should keep this option enabled in unstable and make enableVectors throw an eval warning in unstable now. Once people upgrade to 25.11 they will be seeing the eval warning and be encouraged to migrate. Then we can completely remove this option in 26.05.

Reply:
> make enableVectors throw an eval warning in unstable now

What's the point given the migration happens automatically? The option will be removed eventually and people won't even notice.
We could print a warning if `enableVectorchord` is off.